polkadot/runtime/rococo/src/lib.rs

```diff
@@ -1060,8 +1061,15 @@ impl parachains_slashing::Config for Runtime {
 	type BenchmarkingConfig = parachains_slashing::BenchConfig<200>;
 }
 
+pub type LocationToAccountId<AccountId> = (
```

Comment:
Rococo already has a way of turning locations into accounts, using a different one will definitely cause issues.
You should import it from the `xcm_config` module right next to `lib`.

```rust
use xcm_config::LocationConverter;
```